internal/pkg/githubapi/github_test.go

```diff
@@ -251,3 +252,37 @@ func TestPrBody(t *testing.T) {
 	}
 	assert.Equal(t, string(expectedPrBody), newPrBody)
 }
+
+func TestGhPrClientDetailsGetBlameURLPrefix(t *testing.T) {
+	t.Parallel()
+	tests := []struct {
+		Host  string
+		Owner string
+		Repo  string
```

Comment:
I like the table driven test. Would you add the expected output to the table test as a variable and then assert that values against the value returned by the function under test. This would make the intention of the test easier to see at a glance.